### DISCUSSION TITLE
G7 error code??

### DISCUSSION THREAD

**Discussion by daveneverend12** [NightscoutFoundation/xDrip#3329] [Q&A] [answered]
I need help...dexcom won't replacing my faulty g7 because they said the dont see me connected on their server so don't have the error code.... so what could I do????
....

**Comment by Navid200** [comment]
These are xDrip forums, an open source research tool.
Everyone is welcome to use xDrip at their own risk.

Dexcom is a wonderful company that produces life-changing products for diabetics.  If you use their product, you should follow their guidelines.

It is very easy to incorrectly use xDrip resulting in malfunction.
These are instructions for using G7 with xDrip: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

I believe a G7 Dexcom receiver is considerably cheaper than a G6 receiver.  The easiest way to confirm if something is wrong with G7 is to use the receiver.  If it fails to work with the receiver, it is obvious that something is wrong with G7.  Furthermore, you will be complying with Dexcom requirement.

Do you have a Dexcom receiver?

**Reply by daveneverend12** [reply]
No I don't have it... but dexcom say it won't work...because he think the xdrip work Like the receiver and not like a phone

**Reply by daveneverend12** [reply]
Because dexcom have 2 Chanel...one for receiver and one for phone...

**Reply by Navid200** [reply, marked as answer]
Even if that was true, what I am suggesting is not related.

Let's say you suspect something is wrong with the G7 device you have.
If you have a receiver, you can disable xDrip and set up the receiver to work with your G7.  If it doesn't work, it means something is wrong with your G7.

But, the fact is that xDrip uses the mobile slot of G7 (and G6).  I have gone over this here: https://navid200.github.io/xDrip/docs/Receiver-or-tslim-and-xDrip.html

**Reply by daveneverend12** [reply]
Thanks...I'll go to drugstore to buy it right now... and try it!

**Reply by daveneverend12** [reply]
It work both at the same time ...yeah thanks... so when I have error I just to open my receiver and call dexcom with the error code
